test/WebJobs.Script.Tests/UtilityTests.cs

```diff
@@ -353,5 +354,22 @@ public void IsHttporManualTriggerTests(string triggerType, bool expectedResult)
         {
             Assert.Equal(expectedResult, Utility.IsHttporManualTrigger(triggerType));
         }
+
+        [Theory]
+        [InlineData("createIsolationEnvironment", true)]
+        [InlineData("HttpTrigger2", true)]
+        [InlineData("HttptRIGGER", true)]
+        [InlineData("MANUALtRIGGER", true)]
+        [InlineData("Function-200", true)]
+        [InlineData("hello!", false)]
+        [InlineData("🙅", false)]
+        public void IsValidFunctionNameTests(string functionName, bool expectedResult)
+        {
```

Comment:
Take CurrentCulture as input and add more variations for different cultures.

Reply:
👍